integration-tests/common.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```

Comment:
We could set up github pages for that and do pushes there (this also applies for https://github.com/fabric8-analytics/fabric8-analytics-common/pull/52). We would get https://fabric8-analytics.github.io page. Do you want it?